solidity/random-beacon/test/ReimbursmentPool.test.ts

```diff
@@ -0,0 +1,468 @@
+/* eslint-disable @typescript-eslint/no-unused-expressions */
+
+import { ethers, waffle, helpers } from "hardhat"
+import { expect } from "chai"
+import { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers"
+import { reimbursmentPoolDeployment, params } from "./fixtures"
+import type { ReimbursementPool } from "../typechain"
+
+const ZERO_ADDRESS = ethers.constants.AddressZero
+const { createSnapshot, restoreSnapshot } = helpers.snapshot
+const { provider } = waffle
+const fixture = async () => reimbursmentPoolDeployment()
+
+describe("ReimbursementPool - Pool", () => {
+  let owner: SignerWithAddress
+  let thirdParty: SignerWithAddress
+  let refundee: SignerWithAddress
+  let contractToAuthorize: SignerWithAddress
```

Comment:
```suggestion
  let authorizedContract: SignerWithAddress
```

Reply:
Changed to a more general `thirdPartyContract`, cause `authorizedContract` also won't fit in the `authorize` tests, ie. it's not yet authorized. [b2781e1](https://github.com/keep-network/keep-core/pull/2826/commits/b2781e16230fd4935ec19337a6b5cd5792a60526)